packages/config/config/devices/0x0063/59368_zwa4012.json

```
@@ -0,0 +1,81 @@
+{
+	"manufacturer": "UltraPro",
+	"manufacturerId": "0x0063",
+	"label": "59368 / ZWA4012",
+	"description": "In-Wall Toggle Switch, QFSW, 700S",
+	"devices": [
+		{
+			"productType": "0x4952",
+			"productId": "0x3334",
+			"zwaveAllianceId": 4571
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Single Press",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Double Press",
+			"maxNodes": 5,
+			"isLifeline": true
```

Following the discussion in https://github.com/zwave-js/node-zwave-js/pull/6726, we should remove the group 3 auto association, since the device supports Central Scene.
```suggestion
			"maxNodes": 5
```